addon/services/horizon.js

```diff
@@ -32,7 +36,7 @@ export default Ember.Service.extend({
   currentUser: null,  // set by Horizon Observable
   status: 'init',     // set by Horizon Observable
   hasAuthToken: null, // set by Horizon Observable
-  raw: false,         // Horizon default; specifies detail/structure in watched changes
+  raw: true,          // specifies detail/structure in watched changes (true = RethinkDB changestream)
```

Comment:
We definitely will want the raw updates or we will never know when the initial promise has been resolved, and we will also be pushing the whole state of a collection on every single update. It would work, but be a huge waste.